### ISSUE TITLE
render() function doesn't appear to ever be called before `done` is true

### ISSUE THREAD

**Issue by fabiosantoscode** [denvned/isomorphic-relay-router#44]
Hello!

We've recently updated this library and yesterday we noticed that our beautiful, beautiful spinners were gone!

A quick debug session revealed that the render() function of each route was being called only once, and that was when `done` was true. This means that we never render our spinner :( 

Is there anything that we missed in the transition from `render{Loading,Fetched,Failed}` to just `render`, or is it a bug?

Cheers!! :heart:

**Comment by denvned** [comment]
I am sorry for your spinners. Did you follow the docs: https://github.com/relay-tools/react-router-relay/tree/v0.13.3#render-callback ?

**Comment by denvned** [comment]
Are you spinners not working after route transition? You don't expect them to spin on the initial render, right? The initial render happens only after all the data is loaded.

**Comment by fabiosantoscode** [comment]
Sorry for taking a while, these have been such stressful weeks!

I do not indeed expect them to display in the initial render. That one already has all the data and a full static-html response on the page, so it would be worrysome if they did display :)

We've been investigating some more, and I've started suspecting that it's not okay for `history` to be version 3. I went and updated that because I wanted `history.get` but it may be causing this issue. Still on the case, but that looks really fishy.
